package.json

```diff
@@ -1,21 +1,36 @@
 {
-  "private": true,
-  "name": "vrealize-developer-tools",
-  "displayName": "vRealize Developer Tools",
-  "description": "Provides code intelligence features and enables a more developer-friendly experience when creating VMware vRealize Orchestrator and VMware vRealize Automation content.",
-  "version": "2.2.0",
+  "name": "vmware-pscoe",
```

Comment:
Shouldn't the name be vRealize Developer Tools?

Reply:
Or if this package.json is only for grouping purposes (lerna/yarn workspaces), maybe call it something like vrdt-workspaces?

Reply:
I named it as so. Because the other 2 packages vropkg and vrotsc have namespaces of vmware-pscoe. Just used it for consistancy. The workspace repo is private so it doesn't matter. It is up to you.